build/project-template/internal/nativescript-build.xcconfig

```diff
@@ -1,7 +1,7 @@
 // * NativeScript build related flags
 // * Add [sdk=*] after each one to avoid conflict with CocoaPods flags
 HEADER_SEARCH_PATHS[sdk=*] = $(inherited) "$(SRCROOT)/internal" "$(SRCROOT)/internal/NativeScript/include"
-OTHER_LDFLAGS[sdk=*] = $(inherited) -ObjC -sectcreate __DATA __TNSMetadata "$(CONFIGURATION_BUILD_DIR)/metadata-$(CURRENT_ARCH).bin" -lNativeScript -L"$(SRCROOT)/internal/NativeScript/lib" -licucore -lz -lc++ -framework Foundation -framework UIKit -framework CoreGraphics -framework MobileCoreServices -framework Security
+OTHER_LDFLAGS[sdk=*] = $(inherited) -ObjC -sectcreate __DATA __TNSMetadata "$(CONFIGURATION_BUILD_DIR)/metadata-$(CURRENT_ARCH).bin" -framework NativeScript -F"$(SRCROOT)/internal/NativeScript" -licucore -lz -lc++ -framework Foundation -framework UIKit -framework CoreGraphics -framework MobileCoreServices -framework Security
```

Comment:
We should also copy the framework to the build folder, so it gets included in the application package.